### PR TITLE
Added Ditasic

### PR DESCRIPTION
**Pull request by marcelmat12**
* [x ] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [x ] This PR adds a new recipe.
* [x ] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

**Comment by marcelmat12** [comment]
If possible and theres time pls merge this request :)
Thanks

File: recipes/ditasic/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,38 @@
+package:
+  name: ditasic
+  version: "0.2"
+
+source:
+  url: https://gitlab.com/rki_bioinformatics/ditasic/uploads/9c6dd7f3bfde8d958a03c49ba89276d9/ditasic_0.2.tar.gz
+  md5: 27fbe916e14bf2c47fc215b988dcfb27
+
+build:
+  number: 1
```
Comment (bgruening) [inline recipes/ditasic/meta.yaml]: please start with build-number 0

File: recipes/ditasic/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+  skip: True # [not py3k]
+  rpaths:
+  - lib/R/lib/
+  - lib/
+  
+requirements:
+  host:
+    - python
+    - r-base
+    - r-optparse
```
Comment (bgruening) [inline recipes/ditasic/meta.yaml]: I don't think those host r-dependencies are not needed :(

**Comment by marcelmat12** [comment]
Did the proposed changes!

**Comment by marcelmat12** [comment]
@bgruening did all the proposed changes and added some version requirements, if all is fine pls merge the pull request
Thanks!

File: recipes/ditasic/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+  name: ditasic
+  version: "0.2"
+
+source:
+  url: https://gitlab.com/rki_bioinformatics/ditasic/uploads/9c6dd7f3bfde8d958a03c49ba89276d9/ditasic_0.2.tar.gz
+  md5: 27fbe916e14bf2c47fc215b988dcfb27
+
+build:
+  number: 0
+  skip: True # [not py3k]
```
Comment (corburn) [inline recipes/ditasic/meta.yaml]: `skip: True` can be removed if the  `host` and `run` sections specify `python >=3`. This change will probably cause the linter to suggest adding `noarch: python` to the `build` section.

File: recipes/ditasic/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+  rpaths:
+  - lib/R/lib/
+  - lib/
+  
+requirements:
+  host:
+    - python
+    
+  run:
+    - python >3
```
Comment (corburn) [inline recipes/ditasic/meta.yaml]: `python >=3`

**Comment by marcelmat12** [comment]
I did the proposed chages and now the test fails after i changed to noarch: python
Any tips what to change? @corburn

**Comment by corburn** [comment]
I was wrong about the linter warning. The metaprob error could be resolved by merging/rebasing onto the master branch. I was starting to look into why your original build passed. I did not see any output from the test command `ditasic_matrix.py -h` so I'm guessing it did not run because of the `python >3`. From the following warnings, I am curious what would happen if you included:

```
build:
  '{{ compiler("c") }}
```

```
WARNING (ditasic,opt/ditasic/bin/kallisto): Needed DSO lib/libgcc_s.so.1 found in ['libgcc-ng']
WARNING (ditasic,opt/ditasic/bin/kallisto): .. but ['libgcc-ng'] not in reqs/run, i.e. it is overlinked (likely) or a missing dependency (less likely)
WARNING (ditasic,opt/ditasic/bin/kallisto): did not find - or even know where to look for: /lib64/libm.so.6
WARNING (ditasic,opt/ditasic/bin/kallisto): did not find - or even know where to look for: /lib64/libdl.so.2
WARNING (ditasic,opt/ditasic/bin/kallisto): did not find - or even know where to look for: /lib64/libpthread.so.0
WARNING (ditasic,opt/ditasic/bin/kallisto): did not find - or even know where to look for: /lib64/ld-linux-x86-64.so.2
WARNING (ditasic,opt/ditasic/bin/kallisto): did not find - or even know where to look for: /lib64/librt.so.1
WARNING (ditasic,opt/ditasic/bin/kallisto): did not find - or even know where to look for: /lib64/libc.so.6
WARNING (ditasic,opt/ditasic/bin/mason_simulator): Needed DSO lib/libgcc_s.so.1 found in ['libgcc-ng']
WARNING (ditasic,opt/ditasic/bin/mason_simulator): .. but ['libgcc-ng'] not in reqs/run, i.e. it is overlinked (likely) or a missing dependency (less likely)
WARNING (ditasic,opt/ditasic/bin/mason_simulator): did not find - or even know where to look for: /lib64/libm.so.6
WARNING (ditasic,opt/ditasic/bin/mason_simulator): Needed DSO lib/libbz2.so.1.0 found in ['bzip2']
WARNING (ditasic,opt/ditasic/bin/mason_simulator): .. but ['bzip2'] not in reqs/run, i.e. it is overlinked (likely) or a missing dependency (less likely)
WARNING (ditasic,opt/ditasic/bin/mason_simulator): Needed DSO lib/libgomp.so.1 found in ['libgcc-ng']
WARNING (ditasic,opt/ditasic/bin/mason_simulator): .. but ['libgcc-ng'] not in reqs/run, i.e. it is overlinked (likely) or a missing dependency (less likely)
WARNING (ditasic,opt/ditasic/bin/mason_simulator): did not find - or even know where to look for: /lib64/libpthread.so.0
WARNING (ditasic,opt/ditasic/bin/mason_simulator): Needed DSO lib/libz.so.1 found in ['zlib']
WARNING (ditasic,opt/ditasic/bin/mason_simulator): .. but ['zlib'] not in reqs/run, i.e. it is overlinked (likely) or a missing dependency (less likely)
WARNING (ditasic,opt/ditasic/bin/mason_simulator): did not find - or even know where to look for: /lib64/libc.so.6
WARNING (ditasic,opt/ditasic/bin/mason_simulator): Needed DSO lib/libstdc++.so.6 found in ['libstdcxx-ng']
WARNING (ditasic,opt/ditasic/bin/mason_simulator): .. but ['libstdcxx-ng'] not in reqs/run, i.e. it is overlinked (likely) or a missing dependency (less likely)
```

**Comment by corburn** [comment]
```
requirements:
  # things the build system build should have installed to build the package,
  # but for the most part should not introduce run-time dependencies
  # (e.g. gcc, automake, cmake, ...)
  build:
    - {{ compiler("c") }}
  host:
  # things the build system should have installed to build the package
  # (e.g. python, perl, anything references by the build script)
    - ...
  run:
  # things the user should have installed to run the program
    - ...
```

**Comment by corburn** [comment]
```
package:
  name: ditasic
  version: "0.2"

source:
  url: https://gitlab.com/rki_bioinformatics/ditasic/uploads/9c6dd7f3bfde8d958a03c49ba89276d9/ditasic_0.2.tar.gz
  md5: 27fbe916e14bf2c47fc215b988dcfb27
  
build:
  noarch: python
  number: 0
  rpaths:
  - lib/R/lib/
  - lib/

requirements:
  build:
    - {{ compiler("c") }}
  host:
    - python >=3
  run:
    - python >=3
    - numpy >=1.8.0
    - biopython >=1.70
    - mason >=2.0.7
    - kallisto >=0.43.1

test:
  commands:
    - ditasic_matrix.py -h 

about:
  home: https://rki_bioinformatics.gitlab.io/ditasic/
  license: MIT
  summary: 'DiTASiC is designed as a comprehensive approach for abundance estimation and differential abundance assessment of individual taxa in metagenomics samples.'
```

**Pull request closed without merging**